apps/snitch_core/lib/core/data/schema/stock/stock_movement.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Snitch.Data.Schema.StockMovement do
+  @moduledoc """
+  Model to track inventory movement in location
+  """
+  use Snitch.Data.Schema
+  use Snitch.Data.Schema.Stock
+
+  @type t :: %__MODULE__{}
+
+  schema "snitch_stock_movements" do
+    field(:quantity, :integer, default: 0)
+    field(:action, :string)
```

Comment:
What is an `action`? This can never be set because the `cast` will throw it out.